Sources/AppcuesKit/Presentation/UI/ExperienceData.swift

```diff
@@ -125,17 +165,27 @@ extension ExperienceData {
             switch underlyingValue {
             case .single:
                 underlyingValue = .single(newValue)
-            case .multi(let existingValues):
-                // Toggle the value to be included in the set.
-                underlyingValue = .multi(existingValues.symmetricDifference([newValue]))
+            case .multi(var existingValues, let maxSelections):
+                if existingValues.contains(newValue) {
+                    existingValues = existingValues.filter { $0 != newValue }
+                } else if let maxSelections = maxSelections {
+                    if existingValues.count < maxSelections {
+                        existingValues.append(newValue)
+                    } else {
+                        // Would be selecting more than the max, so no change.
```

Comment:
no issue with whats here, just wondering if we'll need to give any additional UI feedback in the future, in this case of "unresponsive selection"

Reply:
Possibly? Realistically we're leaving it up the the customer to use a label that makes it clear only `n` items can be selected. Otherwise it would be confusing.

Because this is effectively the view model for the control, it'd be pretty straightforward to put something in the `else` block here to trigger some UI feedback.